test/paragraph-cases/embedded-list.html

```diff
@@ -1 +1,2 @@
-<p>hello 1. this should not be a list.</p>
+<p>hello 1. this should not be a list.
+</p>
```

Comment:
Since the test files have a line break at the end of the file, this is preserved in the output.